.gitignore

```diff
@@ -151,3 +151,5 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+#Remove DS_Store: Mac
+**/.DS_Store
```

Comment:
This should go in `$XDG_CONFIG_HOME/git/ignore` ([ref](https://git-scm.com/docs/gitignore), this is ~/.config/git/ignore on Linux by default)